LEGEND.md

```diff
@@ -4,48 +4,88 @@
 
 [2. Slide Shortcuts](#slides)
 
+- [macOS](#macos-slides)
+
+- [Windows](#windows-slides)
+
 [3. Others](#others)
 
+- [macOS](#macos-others)
+
+- [Windows](#windows-others)
+
 ### <a name="mappings">Keyboard Mappings</a>
 
 ![](./assets/img/help_images/web-desktop-keyboard-map.png)
 
-- _Note that the chars on the top require shift + key._
+- _Note that the characters shown at the top of each key require a shift + key._
 
 ### <a name="slides">Slide Shortcuts</a>
 
-#### macOS:
-
-- Cmd + 1 = Waheguru Slide
-- Cmd + 2 = Mool Mantra Slide
-- Cmd + 3 = Empty Slide
-- Cmd + 4 = Anand Sahib (Bhog)
-- Cmd + 5 = Help Guide
-- Cmd + 6 = Shortcut Legend
-- Cmd + / = Jump to search
-
-#### Windows:
-
-- Ctrl + 1 = Waheguru Slide
-- Ctrl + 2 = Mool Mantra Slide
-- Ctrl + 3 = Empty Slide
-- Ctrl + 4 = Anand Sahib (Bhog)
-- Ctrl + 5 = Help Guide
-- Ctrl + 6 = Shortcut Legend
-- Cmd + / = Jump to search
+#### <a name="macos-slides">macOS:</a>:
+
+| Combination | Description                     |
+| ----------- | ------------------------------- |
+| Fn + 1     | Waheguru Slide                  |
```

Comment:
lets keep the casing for the `Fn` button consistent for mac and pc. probably want to make these `fn`